packages/utilities/aria-utils/src/collections/section.ts

```
@@ -6,5 +6,8 @@ import {HTMLNextUIProps, As} from "@nextui-org/system";
  * A modified version of the SectionProps from @react-types/shared, with the addition of the NextUI props.
  *
  */
-export type SectionProps<Type extends As = "div", T extends object = {}> = BaseSectionProps<T> &
-  Omit<HTMLNextUIProps<Type>, "children">;
+export type SectionProps<
+  Type extends As = "div",
+  T extends object = {},
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)